openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -348,6 +348,9 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
 TCN_IMPLEMENT_CALL(jint, SSLContext, free)(TCN_STDARGS, jlong ctx)
```

Comment:
yes but I think I would like to leave it as it is for now to not make it again harder to jump from one version to another for no strong reason